.github/workflows/dependents-report.yml

```diff
@@ -2,6 +2,7 @@ name: Generate top dependents report
 on:
   schedule:
     - cron: '0 14 * * 1'
+  workflow_dispatch:
```

Comment:
is this needed?

Reply:
I figured it would be a valuable option to have when debugging

I could take it out if preferred